app/views/form/check_answers.html.erb

```diff
@@ -24,8 +24,9 @@
 
     <%= form_with model: @case_log, method: "get" do |f| %>
       <%= f.govuk_submit "Save and return to log" do %>
-        <% if @case_log.status == "in_progress" %>
-          <%= govuk_button_link_to "Save and go to next incomplete section", "/logs/#{@case_log.id}/#{@case_log.form.next_incomplete_section_redirect_path(subsection, @case_log)}", secondary: true %>
+        <% next_incomplete_section_redirect_path = @case_log.form.next_incomplete_section_redirect_path(subsection, @case_log) %>
+        <% if @case_log.status == "in_progress" && next_incomplete_section_redirect_path != "error" %>
```

Comment:
Does the method actually return a string "error"?

Reply:
yup. Happy to change, if that's a good use of time

Reply:
Ah I see it literally does, missed that. I think I'd expect `nil` returned but this is fine too.